hazelcast/src/main/java/com/hazelcast/sql/impl/row/JetSqlRow.java

```diff
@@ -81,6 +81,7 @@ public SerializationService getSerializationService() {
         return ss;
     }
 
+    @Nonnull
```

Comment:
Why was this removed?

Reply:
do you mean "added"? I wanted to be explicit with regard to (non)nullability. Some functions in `ExpressionUtils` return null `JetSqlRow`

Reply:
Oh, my bad :laughing: